Launcher.py

```diff
@@ -14,10 +14,11 @@
 import shlex
 import subprocess
 import sys
-from enum import Enum, auto
 from os.path import isfile
 from shutil import which
-from typing import Iterable, Sequence, Callable, Union, Optional
+from typing import Sequence, Union, Optional
+
+from worlds.LauncherComponents import Component, components, Type, SuffixIdentifier
```

Comment:
I believe I wanted to skip that to save on startup time :thinking:
But with apworlds not having EXEs that might be *the* only way? Maybe we should force worlds to do more lazy loading?

Reply:
For now pretty sure it is the only way.
Lazy loading is nice and as we grow becomes ever more nicer. In the future it might be nice to not unload and have the launcher just run functions, rather than another executable that has to load everything again.